DESCRIPTION

```diff
@@ -0,0 +1,7 @@
+Package: htmlwidgetsgallery
+Version: 0.0.0
+Imports: 
+    gh,
+    jsonlite,
+    rlang,
+    yaml
```

Comment:
This DESCRIPTION is needed because `r-lib/actions/setup-r-dependencies` (more precisely, the pak package) requires it to know the dependency packages.